context/app/static/js/pages/Publications/utils.js

```diff
@@ -1,62 +1,25 @@
-function buildAbbreviatedContributors(contributors) {
-  switch (contributors.length) {
-    case 0:
-      return '';
-    case 1:
-      return contributors[0].name;
-    case 2:
-      return contributors.map((contributor) => contributor.name).join(' and ');
-    default:
-      return `${contributors[0].name}, et al.`;
-  }
-}
-
-function buildSecondaryText(contributors, publication_venue) {
-  return [buildAbbreviatedContributors(contributors), publication_venue].filter((str) => str.length).join(' | ');
-}
-
-function buildPublicationPanelProps(publicationHit) {
-  const {
-    _source: { uuid, title, contributors = [], publication_venue, publication_date },
-  } = publicationHit;
-
-  return {
-    key: uuid,
-    href: `/browse/publication/${uuid}`,
-    title,
-    secondaryText: buildSecondaryText(contributors, publication_venue),
-    rightText: `Published: ${publication_date}`,
-  };
-}
-
-function buildPublicationsSeparatedByStatus(publications) {
-  return publications.reduce(
-    (acc, publication) => {
-      const {
-        _source: { publication_status },
-      } = publication;
-
-      if (publication_status === undefined) {
+function countPublicationAggs(aggsBuckets) {
+  return aggsBuckets.reduce(
+    (acc, { key_as_string, doc_count }) => {
+      if (!['false', 'true'].includes(key_as_string)) {
         return acc;
       }
 
-      const publicationProps = buildPublicationPanelProps(publication);
-
-      if (publication_status) {
-        acc.published.push(publicationProps);
-      } else {
-        acc.preprint.push(publicationProps);
-      }
+      /* eslint-disable operator-assignment */
```

Comment:
I'm okay with disabling this eslint rule globally - I find your approach much clearer than trying to use `+=`, clarity is a very "case by case" basis.

We can add this to `.eslintrc.yml` under `rules`: `operator-assignment: [0]`